app/controllers/alchemy/pages_controller.rb

```diff
@@ -104,7 +104,7 @@ def load_index_page
     # @return NilClass
     #
     def load_page
-      @page ||= Page.contentpages.find_by(
+      @page ||= Language.current.pages.find_by(
         urlname: params[:urlname],
         language_code: params[:locale] || Language.current.code
```

Comment:
It might be better readable to have `Site.current.pages` here instead. With that its clearer and one would not think the `language_code` is not needed. The downside would be a bigger cost in terms of sql performance because we are not joining with the Language anymore but the Site so we have a bigger collection to filter (using `language_code`). For a few pages not noticeable for sure, but for hundrets of pages it could be.